### PR TITLE
fix: changed logo in login page

### LINKED ISSUE #954: fix: Change the logo to new logo in LoginActivity
## Summary:

The LoginActivity has the older logo of the app. While the logo is updated now, I think it should be updated in the entire app among which the LoginActivity is missing its logo. 

## Expected design:
Group 19@2x

## Link for asset (logo-outline):
https://drive.google.com/open?id=1y53mJmoZ8lc0YiR9gNtF0V8zZ5muu6mH

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes: #954 
Changed the logo in LoginActivity to the new logo as per the new design.

image

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@rachittt @luckyman20 Please review

**Review by rchtgpt — CHANGES_REQUESTED** [review]
change the name of the logo file from `ic_new_logo.xml` to `ic_logo_outline.xml`

File: mifospay/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -10,30 +10,18 @@
     android:background="@color/colorPrimary"
     android:padding="@dimen/paddingLayout">
 
+
```
Comment (rchtgpt) [inline mifospay/src/main/res/layout/activity_login.xml]: remove this extra line

**Review by rchtgpt — CHANGES_REQUESTED** [review]

File: mifospay/src/main/res/layout/activity_login.xml (hunk 1 above)
Comment (Bhavnaharitsa) [reply mifospay/src/main/res/layout/activity_login.xml]: done

**Comment by Bhavnaharitsa** [comment]
> change the name of the logo file from ic_new_logo.xml to ic_logo_outline.xml

done

File: mifospay/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -12,31 +12,17 @@
...
-        android:id="@+id/textView3"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginVertical="@dimen/value_10dp"
-        android:layout_marginTop="22dp"
-        android:layout_marginBottom="26dp"
-        android:text="@string/mifos_pay"
-        android:textColor="@color/colorAccent"
-        android:textSize="@dimen/textH4"
+        android:layout_height="150dp"
```
Comment (rchtgpt) [inline mifospay/src/main/res/layout/activity_login.xml]: hardcoded dimension

File: mifospay/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -12,31 +12,17 @@
...
+        android:layout_marginBottom="50dp"
```
Comment (rchtgpt) [inline mifospay/src/main/res/layout/activity_login.xml]: hardcoded dimension

**Review by rchtgpt — CHANGES_REQUESTED** [review]

File: mifospay/src/main/res/layout/activity_login.xml (hunk 3 above)
Comment (Bhavnaharitsa) [reply mifospay/src/main/res/layout/activity_login.xml]: Well, I see there are some files in the project which have hardcoded dimensions. Do you want me to make a pr on this issue?

File: mifospay/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -12,31 +12,17 @@
 
     <ImageView
         android:id="@+id/image_logo"
-        android:layout_width="300dp"
-        android:layout_height="160dp"
-        android:src="@drawable/mifos_icon"
```
Comment (devansh-299) [inline mifospay/src/main/res/layout/activity_login.xml:17]: @Bhavnaharitsa we are not using `mifos_icon` anywhere else, so can delete this drawable also

Comment (Bhavnaharitsa) [reply mifospay/src/main/res/layout/activity_login.xml:17]: done

**Review by devansh-299 — APPROVED** [review]

**Review by rchtgpt — APPROVED** [review]
thanks!

**Pull request merged**